static/css/contact-us.less

```diff
@@ -3,10 +3,20 @@
    Contact us page.
    ========================================================================== */
 
-.helpful-terms_header {
-    .h5();
-    margin-right: unit(6px / @font-size, em);
-    vertical-align: middle;
+.helpful-terms {
+    &_header {
+        .h5();
+        margin-right: unit(6px / @font-size, em);
+        vertical-align: middle;
+    }
+
+    &_list {
```

Comment:
Yeah, I started with the `list__horizontal` on the elem but then realized it wouldn't work for larger screen sizes.